net/ddns-scripts/Makefile

```diff
@@ -70,6 +70,18 @@ define Package/ddns-scripts-cloudflare/description
 endef
 
 
+define Package/ddns-scripts-gcp
+  $(call Package/ddns-scripts/Default)
+  TITLE:=Extension for Google Cloud DNS API v1
+  DEPENDS:=ddns-scripts +curl +openssl-util
+  PROVIDES:=ddns-scripts_gcp_v1
```

Comment:
This is a new provider so this PROVIDES is not necessary. We did not changed the name.